### DISCUSSION TITLE
다른 월드로 이동이 가능한가요?

### DISCUSSION THREAD

**Discussion by koldshine** [naverz/zepeto-studio-kor#772] [5. Q&A]
안녕하세요.
유니티로 월드 개발중인데요
다른 월드(방)로 이동하는 api나 방법이 있을까요?

**Comment by minjujuu** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/607
현재 다른 월드로의 이동은 지원하지 않는다고 합니다.

**Reply by koldshine** [reply]
감사합니다
